spec/services/track_visit_service_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../unit_helper'
+require_relative '../../services/track_visit_service'
+require_relative '../../models/page'
+require_relative '../../models/page_visit'
+
+describe TrackVisitService do
+  it 'creates an entry in PageVists table' do
+    page = Page.create(target_url: 'http://other-test.com')
+
+    prev_count = PageVisit.count
+    TrackVisitService.new(page).track
+    _(PageVisit.count).must_equal(prev_count + 1)
```

Comment:
```ruby
page = Page.create(target_url: 'http://other-test.com')
page_visit = TrackVisitService.new(page).track
_(page_visit.new?).must_equal(false)
_(page_visit.page_id).must_equal(page.id)
```

I think the current implementation is ok, but I think that way we check it completely: if we created a new visit record and if it is associated with this page.